Frontend/ui-customer/src/Flow.purs

```diff
@@ -392,6 +392,7 @@ accountSetUpScreenFlow = do
 homeScreenFlow :: FlowBT String Unit
 homeScreenFlow = do
   (GlobalState currentState) <- getState
+  _ <- pure $ hideKeyboardOnNavigation true
```

Comment:
did you check this change in search location model if you do like this keyboard will close for every input